### PR TITLE
In the case of multiple network cards, a valid host is returned

### PR DESCRIPTION
**Pull request by CalvinKirs**
## Brief change log
In the case of multiple network cards, a valid host is returned

## Verify this pull request

This pull request is already covered by existing tests, such as *(please describe tests)*.

This change added tests and can be verified as follows:

*(example:)*

  - *Added dolphinscheduler-dao tests for end-to-end.*
  - *Added CronUtilsTest to verify the change.*
  - *Manually verified the change by testing locally.*

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Find first valid IP from local network card
+     *
+     * @return first valid local IP
+     */
+    public static synchronized InetAddress getLocalAddress() {
+
+        if (null != LOCAL_ADDRESS) {
+            return LOCAL_ADDRESS;
```
Comment (gabrywu) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java:76]: what's purpose of this variable LOCAL_ADDRESS ?

Comment (CalvinKirs) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java]: > what's purpose of this variable LOCAL_ADDRESS ? never assigned a value

Sorry, this is my oversight, I have completed the change, please review it

**Comment by dailidong** [comment]
@gabrywu , could you review this PR ?

**Comment by dailidong** [comment]
good job

**Pull request merged**